.rubocop.yml

```diff
@@ -76,3 +83,12 @@ Metrics/BlockLength:
     Exclude:
       - 'spec/**/*'
       - 'config/routes.rb'
+
+Layout/EmptyLinesAroundAccessModifier:
+    Enabled: false
+
+Layout/SpaceInsideBlockBraces:
+    Enabled: false
+
+Layout/IndentationWidth:
+    Enabled: false
```

Comment:
It would be good to configure your editor so that it always add lines at the end of the file in order to avoid noise in the review. Github also suggests this approach with the symbol as a warning

Reply:
Currently, in the codebase, there are 69 offences relating to this. Since we've lived with this so far I assumed its safe to disable

![image](https://user-images.githubusercontent.com/22743709/67762437-5656f800-fa3d-11e9-8b55-ad7a52608eea.png)